components/list/demo/list-demo-control.js

```diff
@@ -38,6 +41,17 @@ class ListDemoControl extends LitElement {
 		`];
 	}
 
+	connectedCallback() {
+		super.connectedCallback();
+		const list = document.querySelector(`${this.target} d2l-list`);
+		list.addEventListener('change', this._onListChange.bind(this));
```

Comment:
I know this is just a demo component, but something to be mindful of: `removeEventListener` will only remove listeners if it is exactly the same instance of the function.  The `bind()` creates a new function, so in this case, the instances of those functions that are being passed into `addEventListener` and `removeEventListener` are different.  In cases where we need to use `addEventListener`, it is best to do the `bind` in the `constructor`,
```
constructor() {
   super();
   this._onListChange = this._onListChange.bind(this);
}
```
That way everything is referencing the same instance of the function.  :)

Reply:
Thank you for pointing this out. Honestly, I was worried this might be the case and was never really sure. I should have just asked but will update this since other may copy it.

- [ ] Update to remove the listener properly as said in the comment above